ballerina/request_processor.bal

```diff
@@ -51,6 +51,29 @@ isolated function processSubscriptionVerification(http:Caller caller, http:Respo
     }
 }
 
+isolated function processUnsubscriptionVerification(http:Caller caller, http:Response response, 
+                                                    RequestQueryParams params, HttpToWebsubAdaptor adaptor) {
+    UnsubscriptionVerification message = {
+        hubMode: <string>params?.hubMode,
```

Comment:
What if hubMode is not there ? Will it result in panic ?

Reply:
When the `RequestQueryParams` passed here, `hubMode` will be already checked. Hence logically there would not be any `panic` here.